Tests/Regression/Agent/CommandLine_Bugs.robot

```diff
@@ -54,3 +54,20 @@ Run agent with -x (xml mode)
 	${result}= 	Query Result DB 	${dbfile} 	Select count(*) from ResultSummary;
 	Should Be True	${result[0][0]} > 0
 	Should Be Equal As Numbers	${result[0][0]} 	4
+
+Check If The Not Buildin Modules Are Included In The Agent Setup File
+	[Tags]	windows-latest	ubuntu-latest	macos-latest	Issue #123
+	${imports}	Get Modules From Program .py File That Are Not BuildIn
+	...    ${CURDIR}..${/}..${/}..${/}..${/}rfswarm_agent${/}rfswarm_agent.py	6
+
+	Log	${imports}
+
+	${requires}	Get Install Requires From Setup File
+	...    ${CURDIR}..${/}..${/}..${/}..${/}setup-agent.py
+
+	Log	${requires}
+
+	FOR  ${i}  IN  @{imports}
+		Should Contain	${requires}	${i}
+		...    msg="Some modules are not in manager setup file"
+	END
```

Comment:
Potential issue here is it will only identify the first missing dependency not all of them is there are more than 1, probably this is OK but it means re-running the test to find the second one and again for the 3rd (hopefully that won't ever be an issue)

Reply:
I'm not too worried about this one for now, it's unlikely to happen in this situation, just pointed it out for your learning